File: sdk/java-sdk/src/test/java/com/evernym/verity/sdk/protocols/presentproof/v0_6/RestrictionTest.java
package com.evernym.verity.sdk.protocols.presentproof.v0_6;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class RestrictionTest {

    @Test
    public void buildTest() {
        Restriction test = RestrictionBuilder.blank()
                .schemaId("ASDFASDF")
                .build();

        assertEquals(test.toJson().toString(), "{\"schema_id\":\"ASDFASDF\"}");
    }

}

File: sdk/dotnet-sdk/src/VeritySDK.Test/Protocols/PresentProof/RestrictionTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using VeritySDK.Protocols.PresentProof;

namespace VeritySDK.Test
{
    [TestClass]
    public class RestrictionTest
    {
        [TestMethod]
        public void buildTest()
        {
            Restriction test = RestrictionBuilder.blank()
                    .schemaId("ASDFASDF")
                    .build();

            Assert.AreEqual(test.toJson().ToString(), "{\"schema_id\": \"ASDFASDF\"}");
        }
    }
}
